### PR TITLE
Feature/hello

### PR DESCRIPTION
**Pull request by Obote**
## What does this PR do?
- Setup git workflows 
- Create  Hello-Microverse app

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (teekaytech) [inline .gitignore:1]: - [ ] You have to list the `test.md` inside the `.gitignore` file. This is one of the requirements of this project that must be satisfied.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Obote :wave:

__This is a brilliant effort from you :+1: You've really done an amazing job so far. In this PR, my major highlights include__

- Your linters are setup correctly and no errors are reported :100:  
- Your PR title, summary and commit messages are descriptive :+1: 

__However, you need to make some changes before moving on to the next assignment. Please check my comments below and fix them as appropriate.__

### **Required Changes** :recycle: 

- [ ] It is good that you've used the Readme template. However, you need to use the actual data for each of the sections of your Readme. The placeholders you're having on your Readme are there to guide you on what should be added to each of the sections. Replacing the placeholders with the project documentation in the Readme will guide your users on what the project is about, how to use it, etc. 
- [ ] For this project, GitHub flow is the recommended flow: Pull Request should be made from the “feature branch” (e.g. `feature/hello`)  into the `main/master` branch. You've used the `develop` branch as your base branch instead of the `master` branch. Kindly change it.
- [ ] Check the comment under the review.

### **Optional Changes** :zap: 
N/A


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Comment by Obote** [comment]
## Changes Made
#Hello @teekaytech 
# Thank you for your review feedback, I have made changes as requested. They include,
Replaced placeholder with project documentation.
Pull Request made from "feature/hello" into the main branch.
Included test.md in gitignore file.

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @Obote  , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**